components/learn/modules/ROOT/pages/upgrading-smart-contracts.adoc

```diff
@@ -487,6 +487,24 @@ To test upgradeable contracts we should create unit tests for the implementation
 
 When we want to upgrade, we should create unit tests for the new implementation contract, along with creating higher level tests for testing interaction via the proxy after we upgrade using `upgradeProxy`, checking that state is maintained across upgrades.
 
+== Possible issues
+
+While learning how to upgrade contract you might find yourself in a situation of conflicting contract on the local environment.
```

Comment:
What do you mean by "conflicting contract"?

Reply:
When all deploy scripts are run if the `npx hardhat node` is not restarted it wont allow to deploy all contracts

Reply:
@frangio also in some cases `npx hardhat clean` needed to be run